### DISCUSSION TITLE
03/29 Daily Spring

### DISCUSSION THREAD

**Discussion by revature-ives** [revature-ives/PickandPayStore#8] [Daly Spring]
_**What Idid Yesterday**_

Create the project in GitHub and assign collaborators
**Obstacles**
Organize a workflow
**What I gonna do tomorrow**
Assign task to the collaborators

**Comment by revature-ives** [comment]
@Costin79 @mattparker412 @sfrazier851  please write you daily spring

**Comment by Costin79** [comment]
What I did Yesterday
Fixed issues with the SwiftUI cart component.
Obstacles
Today I need to integrate the SwiftUI swopping cart component into the main Story Board app
What I gonna do tomorrow
Start working on a front store SwiftUI / Story Board component
